content/ch-algorithms/teleportation.ipynb

```diff
@@ -24415,7 +24485,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.3"
+   "version": "3.6.9"
```

Comment:
you should probably revert the change on this line right before you check it in?

Reply:
I don't think this makes much difference. Thanks